package.json

```diff
@@ -45,11 +45,13 @@
     "flow": "flow src",
     "flowgen": "flowgen src/index.d.ts -o src/index.js.flow",
     "flowtyped": "flow-typed install",
-    "format": "prettier README.md CONTRIBUTING.md CODE_OF_CONDUCT.md './src/**/*.{ts,tsx}' src/index.d.ts --write",
+    "format": "yarn format-js && yarn format-ios",
+    "format-ios": "find apple/ -iname *.h -o -iname *.m -o -iname *.cpp -o -iname *.mm | xargs clang-format -i",
+    "format-js": "prettier README.md CONTRIBUTING.md CODE_OF_CONDUCT.md './src/**/*.{ts,tsx}' './Example/**/*.{ts,tsx}' src/index.d.ts --write",
```

Comment:
You could do *.md to future prevent the script whenever new .md files would are added to the repo.